src/main/java/uk/ac/ebi/eva/pipeline/jobs/steps/tasklets/PopulationStatisticsGeneratorStep.java

```diff
@@ -75,4 +82,66 @@ public RepeatStatus execute(StepContribution contribution, ChunkContext chunkCon
 
         return RepeatStatus.FINISHED;
     }
+
+    public static URI getStatsBaseUri(String outputDirStatistics, String studyId, String fileId) throws URISyntaxException {
```

Comment:
Wouldn't it be more convenient to have higher level methods in the `URLHelper` class to support this and the other `getXXXStatsURI` operations?

The existing, general `createUri` method would stay for arbitrary paths, but we could have predefined methods for statistics and annotation (integrating the functionality from `VepUtils` too).